### PR TITLE
LPS-109391 SF

### PR DESCRIPTION
**Pull request by ambrinchaudhary**

### CODE REVIEW

**Comment by 4lejandrito** [comment]
Just started reviewing :)

File: modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/page.jsp [hunk 1]
```diff
@@ -19,12 +19,35 @@
...
+>
+	<link href="<%= PortalUtil.getStaticResourceURL(request, application.getContextPath() + "/css/main.css") %>" rel="stylesheet" type="text/css" />
+</liferay-util:html-top>
+
+<c:choose>
+	<c:when test="<%= type.equals(RatingsType.LIKE.getValue()) %>">
+		<clay:button
+			elementClasses="btn-outline-borderless btn-outline-secondary btn-sm"
+			icon="heart"
+		/>
```
Comment (boton) [inline modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/page.jsp]: Move CSS inside the conditional

```suggestion
<c:choose>
	<c:when test="<%= type.equals(RatingsType.LIKE.getValue()) %>">
	        <liferay-util:html-top
	            outputKey="ratings-taglib"
                >
	           <link href="<%= PortalUtil.getStaticResourceURL(request, application.getContextPath() + "/css/main.css") %>" rel="stylesheet" type="text/css" />
                </liferay-util:html-top>
		<clay:button
			elementClasses="btn-outline-borderless btn-outline-secondary btn-sm"
			icon="heart"
		/>
```

**Comment by 4lejandrito** [comment]
Pull request submitted to https://github.com/boton/liferay-portal/pull/137. [See changes here.](https://github.com/4lejandrito/liferay-portal/compare/d5ca70e05a4d~5...d5ca70e05a4d)

**Comment by 4lejandrito** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/85666. [See changes here.](https://github.com/4lejandrito/liferay-portal/compare/d5ca70e05a4d~5...d5ca70e05a4d)

**Pull request closed without merging**